packages/default/scss/popup/_theme.scss

```diff
@@ -23,14 +24,14 @@ $list-container-no-data-text: rgba( $text-color, .5 ) !default;
         box-shadow: $list-container-shadow;
     }
     .k-popup .k-list .k-item > .k-group {
-        background: $text-color;
-        color: $bg-color;
+        color: $grouping-header-text;
+        background: $grouping-header-bg;
     }
     .k-popup .k-list .k-item > .k-group::before {
-        border-color: $text-color $text-color transparent transparent;
+        border-color: $grouping-header-bg $grouping-header-bg transparent transparent;
     }
     .k-group-header+div > .k-list > .k-item.k-first::before {
-        border-color: $border-color;
+        border-color: $grouping-border;
```

Comment:
The `$grouping-border` default value is currently set to `$grouping-header-bg`, which refers to `$text-color`.
This changes the previous code that uses `$border-color`. Is this intentional?

Reply:
the issue is fixed